ui/src/buckets/selectors/index.ts

```diff
@@ -5,28 +5,36 @@ export const SYSTEM = 'system'
 
 export const isSystemBucket = (type: string): boolean => type === SYSTEM
 
-export const getSortedBucketNames = (buckets: Bucket[]) =>
-  buckets
-    .sort((a, b) => {
```

Comment:
funny thing about JS, each browser implements `sort` just a bit different, maddening at times. Wonder if this was passing in some js runtimes and not others 🤔

Reply:
it was actually passing locally but not on circleci. I completely thought that the sort method would work deterministically (especially when it came to two values being compared to one another in a particular order) but I guess not.

:sad_trombone: